src/java.base/share/classes/java/util/HexDigits.java

```diff
@@ -105,14 +105,14 @@ private HexDigits() {
     /**
      * Combine two hex shorts into one int based on big endian
      */
-    static int digit(int b0, int b1) {
+    static int packDigits(int b0, int b1) {
         return (DIGITS[b0 & 0xff] << 16) | DIGITS[b1 & 0xff];
     }
 
     /**
      * Combine four hex shorts into one long based on big endian
      */
-    static long digit(int b0, int b1, int b2, int b3) {
+    static long packDigits(int b0, int b1, int b2, int b3) {
```

Comment:
Same comment here:

```java
/**
 * Return a big-endian packed long for the 8 ASCII bytes for an input unsigned int value.
 */
static int packDigits32(int b) {
    return (((long) DIGITS[(b >> 24) & 0xff]) << 48)
            | (((long) DIGITS[(b >> 16) & 0xff]) << 32)
            | (DIGITS[(b >> 8) & 0xff] << 16)
            | DIGITS[b & 0xff];
}
```
And use sites become:
```java
// old
HexDigits.packDigits((int) (msb >> 56), (int) (msb >> 48), (int) (msb >> 40), (int) (msb >> 32))
// new
HexDigits.packDigits32((int) (msb >> 32))
```
and `HexDigits.packDigits32((int) (lsb >> 16))`

Reply:
but it's return a long value

Reply:
My fault, fixed the return type in my code snippet.